src/components/elements/table/GenericTable.tsx

```diff
@@ -85,9 +85,21 @@ export interface Props<T> {
   belowRowsContent?: ReactNode; // component to insert below all rendered rows, above footer
 }
 
+export const stickyCellClassName = 'StickyTableCell';
+
+// primary.100 on top of paper bg, so we don't layer semi-transparent primary.100 twice
+const nonTransparentPrimary100bg = (theme: Theme) =>
+  `linear-gradient(${theme.palette.primary[100]}, ${theme.palette.primary[100]}),${theme.palette.background.paper};`;
```

Comment:
this is 😭  but it was the only fix I could find, primary 100 is semi-transparent so we couldn't use the previous approach of just setting the background color to make sticky cells non-transparent. Instead here I use linear-gradient to layer the primary.100 over the background paper. @martha

Reply:
seems fine to me! Thanks :)